apps/ui/src/networks/offchain/constants.ts

```diff
@@ -1,9 +1,20 @@
 export const AUTHS = {};
-export const PROPOSAL_VALIDATIONS = {};
+export const PROPOSAL_VALIDATIONS = {
+  any: 'Any',
+  basic: 'Basic',
+  'passport-gated': 'Passport gated',
+  arbitrum: 'Arbitrum',
+  'karma-eas-attestation': 'Karma EAS Attestation',
+  'only-members': 'Only members'
+};
```

Comment:
We should organise it in sx.js strategies soon, can you create GH issue to track this? If you want I can take this task.

I assume that `any` means anyone can propose, `only-members` use the list and for the rest we call `validate`? I'd create entrypoint like we have there:
https://github.com/snapshot-labs/sx-monorepo/blob/a0a0aecc1158d55fb6d3fb88138b2f6caa2636ab/packages/sx.js/src/strategies/evm/index.ts#L14-L35

For all of those (+ default for voting power strategy), I assume that logic in `basic`, `passport-gated`, `arbitrum`, `karma-eas-attestation` would be the same so it would be good to make it generic so we can do:
```js
if (name === 'any') return createAnyProposalValidationStrategy();
if (name === 'passport-gated') return createRemoteValidationStrategy(name);
if (name === 'arbitrum') return createRemoteValidationStrategy(name);
if (name === 'karma-eas-attestation') return createRemoteValidationStrategy(name);
if (name === 'default') return createDefaultStrategy();
```

Reply:
I'll let you take case of this, since you have a better understanding of the whole situation

Reply:
I will work on this once this PR is merged: https://github.com/snapshot-labs/sx-monorepo/issues/74